specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2024-09-10-preview/HybridCompute.json

```diff
@@ -2771,6 +2966,146 @@
       },
       "description": "Describes Extension Metadata properties"
     },
+    "ExtensionValueListResultV2": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ExtensionValue"
```

Comment:
This should be ExtensionValueV2 rather than ExtensionValue, shouldn't it? This collection response is supposed to be an array of items the same type as the point response. The mismatch between ExtensionValue and ExtensionValueV2 is what's causing the GetCollectionResponseSchema check to fail.

Reply:
If this is correct, good catch, and an example why it's important to not assume errors are false positives until you have thoroughly investigated.

Reply:
This should be ExtensionValue and not ExtensionValueV2. ExtensionValueV2 has additional properties that we do not want to expose in ExtensionValueListResultV2, which is the response of paths["/providers/Microsoft.HybridCompute/locations/{location}/publishers/{publisher}/extensionTypes/{extensionType}/versions"].get

GetCollectionResponseSchema check is failing from an existing path, not this one